### PR TITLE
added json configs for HD coldbox

### PR DESCRIPTION
**Pull request by glehmannmiotto**
added a sub-directory with hardware map, pinning file and json configs for daq, card controller and wibs.

### CODE REVIEW

File: config/hd_coldbox_tests/flx_card_hd_coldbox.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+    "flxcardcontroller": {
+        "hardware_map_file": "HardwareMap_VST.txt",
```
Comment (wesketchum) [inline config/hd_coldbox_tests/flx_card_hd_coldbox.json]: This looks like the wrong hardware map file name?

File: config/hd_coldbox_tests/daq_hd_coldbox.json [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+    "mlt_send_timed_out_tds": false,
+    "mlt_max_td_length_ms": 1000
+  },
+  "dataflow": {
+    "host_dfo": "np04-srv-012.cern.ch",
+    "apps": [
+      {
+        "app_name": "dataflow0",
+        "token_count": 10,
+        "output_paths": [ "/data0/test", "/data1/test/" ],
```
Comment (wesketchum) [inline config/hd_coldbox_tests/daq_hd_coldbox.json]: why are there two output paths defined here?

**Review by wesketchum — COMMENTED** [review]
I think it looks good overall: noted two places where things seemed wrong. Would replace https://github.com/DUNE-DAQ/daq-systemtest/pull/15. 

If these were the exact configs run in coldbox tests, please indicate so and that it worked.

File: config/hd_coldbox_tests/flx_card_hd_coldbox.json (hunk 1 above)
Comment (glehmannmiotto) [reply config/hd_coldbox_tests/flx_card_hd_coldbox.json]: this is a bug indeed! fixing it now.

File: config/hd_coldbox_tests/daq_hd_coldbox.json (hunk 2 above)
Comment (glehmannmiotto) [reply config/hd_coldbox_tests/daq_hd_coldbox.json]: this was just to be able to push up rate and fragment size without being limited by disk writing.

**Review by wesketchum — APPROVED** [review]

**Pull request merged**